src/apps/user/services.py

```diff
@@ -41,40 +43,34 @@ def register_user(session: Session, user: UserRegisterSchema) -> UserOutputSchem
 
     return UserOutputSchema.from_orm(new_user)
 
-
 def authenticate(username: str, password: str, session: Session) -> User:
-    user = session.execute(select(User).filter(username == User.username)).scalar()
+    user = session.scalar(select(User).filter(username == User.username).limit(1))
     if not (user or passwd_context.verify(password, user.password)):
-        raise auth_exception
+        raise AuthException("Invalid Credentials")
     return user
 
-
 def get_single_user(session: Session, user_id: int) -> UserOutputSchema:
-    user_object = session.execute(select(User).filter(User.id==user_id)).scalar()
-    if not user_object:
-        raise user_does_not_exist_exception
+    if not (user_object := if_exists(User, "id", user_id, session)):
+        raise DoesNotExist(User.__name__, user_id)
 
     return UserOutputSchema.from_orm(user_object)
 
+def get_all_users(session: Session, page_params: PageParams) -> PagedResponseSchema:
+    instances = session.execute(select(User))
```

Comment:
```suggestion
    query = select(User)
```
Replace like this everywhere and gitara siema